Gemfile

```diff
@@ -109,6 +109,7 @@ gem 'roo', '~> 2.8.0'
 gem 'friendly_id', '~> 5.2.4'
 
 gem 'papercrop', github: 'agilesix/papercrop', ref: 'turbolink-fix'
```

Comment:
⚠️ Is it OK to remove/uninstall the papercrop gem now?

Reply:
if we're not using it, yes please~ :smile:

Reply:
removed! thank you :)